py/dml/codegen.py

```diff
@@ -104,18 +104,26 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         assert LoopContext.current is self
         LoopContext.current = self.prev
+    def break_(self, site):
+        raise ICE(site, f'{__class__.__name__}.break_ not implemented')
+    def cont_(self, site):
+        raise ECONTU(site)
```

Comment:
Motivation for having `break_` ICE but not `cont_`: `ECONTU` exists. `EBREAKU` doesn't.

Reply:
would `@abstractmethod` be an option?

Reply:
It would.